elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreIntegrationTest.java

```diff
@@ -226,6 +226,114 @@ public void whereFilterTest() throws Exception {
         runQueryWithExpectedResult(graphQLRequest, expected);
     }
 
+    @Test
+    public void sortingTest() throws Exception {
```

Comment:
Can we also add a test to the SQLQueryEngineTest class as well?  These tests are great - but I'd like to have full coverage for proper SQL generation at the UT level.